client/src/components/ComboBox/ComboBox.scss

```diff
@@ -111,8 +111,8 @@ $spacing-sm: theme('spacing.5');
   grid-column: 2 / span 1;
 }
 
-.w-combobox__option-icon {
-  color: theme('colors.icon-secondary');
+.w-combobox__option-icon,
+.w-combobox-preview__icon {
```

Comment:
Not sure if it's OK to have the styles for the preview component here, but I needed the same styles so I figured I'd just use them